index.html

```diff
@@ -19,7 +19,7 @@
 	<script type="text/javascript" src="js/prefixfree.min.js"></script>
 </head>
 <body>
-<h1><a href="http://libreprojects.net"><img src="images/favicon-touch.png" alt="" /><span>Libre Projects</span></a></h1>
+<h1><a href="http://libreprojects.net"><img src="images/favicon-touch.png" alt="libreprojects logo" /><span>Libre Projects</span></a></h1>
```

Comment:
This is unnecessary, because the logo is just cosmetic. The empty alt tag is completely fine, because inside the link it already says the name of the site. :)